src/components/common/ShareStructure.vue

```diff
@@ -316,6 +321,9 @@ export default class ShareStructure extends Mixins(CurrencyLookupMixin) {
         const name = this.shareStructure.name
         this.shareStructure.name = name.substr(0, name.indexOf(' Shares'))
       }
+
+      // validate initial data
+      this.$nextTick(() => this.$refs.shareStructureForm.validate())
```

Comment:
This immediately shows any errors when a share class with invalid par value is edited.